config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
+    ->withPhpSets(
+        true
+    )
+    ->withSets([
+        // Rector sets
+
+        LevelSetList::UP_TO_PHP_73,
+        // LevelSetList::UP_TO_PHP_74,
+        // LevelSetList::UP_TO_PHP_80,
+        // LevelSetList::UP_TO_PHP_81,
+        // LevelSetList::UP_TO_PHP_82,
+        // LevelSetList::UP_TO_PHP_83,
+
+        // SetList::CODE_QUALITY,
+        // SetList::CODING_STYLE,
+        // SetList::DEAD_CODE,
+        // SetList::EARLY_RETURN,
+        // SetList::INSTANCEOF,
+        // SetList::NAMING,
+        // SetList::PRIVATIZATION,
+        SetList::STRICT_BOOLEANS,
+        SetList::TYPE_DECLARATION,
+
+        // PHPUnit sets
+
+        PHPUnitSetList::PHPUNIT_90,
+        // PHPUnitSetList::PHPUNIT_100,
+        // PHPUnitSetList::PHPUNIT_CODE_QUALITY,
+    ])
+    ->withRules([
+        AddVoidReturnTypeWhereNoReturnRector::class,
+    ])
+    ->withImportNames(true, true, false)
+    ->withSkip([
+    ]);
```

Comment:
[I don't think we need this if we're not skipping anything](https://getrector.com/documentation/ignoring-rules-or-paths).

Reply:
Done.